catalyst/dl/experiment/config.py

```diff
@@ -290,14 +348,23 @@ def get_loaders(self, stage: str) -> "OrderedDict[str, DataLoader]":
 
         return loaders
 
+    @staticmethod
+    def _get_callback(**params):
+        wrapper_params = params.pop("_wrapper", None)
+        callback = CALLBACKS.get_from_params(**params)
+        if wrapper_params:
+            wrapper_params["base_callback"] = callback
+            return ConfigExperiment._get_callback(**wrapper_params)
+        return callback
```

Comment:
what do you think about another approach, like here https://github.com/catalyst-team/catalyst/blob/master/catalyst/contrib/optimizers/lookahead.py#L84 ?
so in the config it should looks like:
```
callback_name:
    callback: WrapperCallback
    param_1: ...
    param_2: ...
    base_callback_params:
        param_3: ...
        param_4: ...

```

Reply:
I think my solution needs 1 line in config.yml compared to 2 lines here